pkg/etcd/etcd.go

```diff
@@ -54,6 +55,10 @@ const (
 	defaultKeepAliveTimeout = 10 * time.Second
 
 	maxBackupRetention = 5
+
+	MasterLabel       = "node-role.kubernetes.io/master"
+	ControlPlaneLabel = "node-role.kubernetes.io/control-plane"
+	EtcdRoleLabel     = "node-role.kubernetes.io/etcd"
```

Comment:
We already have these as consts at https://github.com/k3s-io/k3s/blob/699ea165231cf6d665425c50ff5c75e1831f4afb/pkg/server/server.go#L42-L44